assets/_scss/components/_forms.scss

```diff
@@ -129,3 +148,14 @@ input.usa-input-medium {
   font-style: italic;
   font-weight: normal;
 }
+
+.usa-input-buttons_inline {
```

Comment:
@msecret do we want to a dash rather than an underscore here since inline a modifier?